setup.py

```diff
@@ -13,13 +13,16 @@
     author_email="dmach@redhat.com",
     license="MIT",
     install_requires=[
-        "pyxdg"
+        "pyxdg",
+        "productmd",
+        "six"
```

Comment:
This is a dependency for `productmd`, should be removed when release-engineering/productmd#38 is closed.